apps/prairielearn/python/zygote_utils_test.py

```diff
@@ -44,7 +44,7 @@ def test_all_integers_within_limits_no_exception(item: Any) -> None:
     try:
         zu.assert_all_integers_within_limits(item)
     except Exception as err:
-        assert False, err
+        raise AssertionError() from err
```

Comment:
I don't quite get what's going on here; why not just call `zu.assert_all_integers_within_limits` directly since that will raise its own exception?

Reply:
Yeah... (linter passes != sanity check passes) 😄

Reply:
Added comment and changed code in cb0dd63df